catalog/ui/catalog-ui-search/src/main/webapp/component/list-editor/list-editor.view.js

```diff
@@ -56,6 +60,37 @@ module.exports = Marionette.LayoutView.extend({
       })
     );
   },
+  handleListTemplate() {
+    this.$el.toggleClass('is-template', this.listTemplateId !== 'custom');
+  },
+  showListTemplate() {
+    if (this.options.showListTemplates === true) {
+      const propertyModel = new Property({
+          label: 'Template',
+          value: [this.listTemplateId],
+          enum: [{
+            label: 'Custom',
+            value: 'custom'
+          }].concat(properties.listTemplates.map((template) => {
+            return {
+              label: template.id,
+              value: template.id,
+              class: List.getIconMapping()[template['list.icon']]
+            };
+          })),
```

Comment:
:pencil2: This function syntax can be simplified. Curly braces can be indicated as object delimiters rather than function body delimiters by using parentheses like in the code below:
```
template => ({
        label: template.id,
        value: template.id,
        class: List.getIconMapping()[template['list.icon']]
})
```
Also, the parentheses around `template` are unnecessary.

Reply:
❓  Do we have a style guide that provides authority on stuff like this?